### PR TITLE
jiratools: make config example more general use

### PR DESCRIPTION
**Pull request by ecedmondson**
Follow up to PR #6  and to Issue #7.

### CODE REVIEW

File: jiratools/jira.config.example [hunk 1]
```diff
@@ -6,9 +6,9 @@ PASSWORD=YOUR_PASSWORD
 DEFAULT_ASSIGNEE=%(USERNAME)s
 TEST_PROJECT=YOUR_JIRA_PROJECT
 DEFAULT_ISSUE_TYPE=Story
-DEFAULT_DESCRIPTION=Do necessary QE testing around the linked Dev JIRA
+DEFAULT_DESCRIPTION=Linking a new story to {dev_jira_id} using jiratools cli.
```
Comment (bradsbrown) [inline jiratools/jira.config.example]:
```suggestion
DEFAULT_DESCRIPTION=Do necessary work related to linked JIRA.
```

Comment (bradsbrown) [reply jiratools/jira.config.example]: Note that, as it currently stands, no formatting values are passed to the `description` field -- it's not used as a template, just a string. To fill in values, you'd need to update the code behidn it as well.

**Review by bradsbrown — CHANGES_REQUESTED** [review]
One note on the `DEFAULT_DESCRIPTION` change.

**Comment by ecedmondson** [comment]
I can't assign in this repo so I'm tagging in @bradsbrown

**Review by bradsbrown — APPROVED** [review]
LGTM

**Pull request merged**